release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2813,7 +2813,7 @@ $ oc adm release info 4.15.10 --pullspecs
 
 * Previously, clusters created before {product-title} 4.7 will have had signer keys for api-int endpoint updated unexpectedly when users upgraded to {product-title} 4.15 due to the installer deleting the SecretTypeTLS and then recreating the secret with `kubernetes.io/tls` type.  With this release, the issue is resolved by the installer changing the secret type without deleting the secret. (link:https://issues.redhat.com/browse/OCPBUGS-31807[*OCPBUGS-31807*])
 
-* Previously, when users imported image stream tags, ICSP (In-Circuit Serial Programming) was not allowed to co-exist with IDMS (Integrated Database Managing System) and ITMS (ImageTagMirrorSet). OpenShift ignored any IDMS/ITMS created by the user and favored ICSP. With this release, they are allowed to co-exist as importing image stream tags will now respect IDMS/ITMS when ICSP is also present. (link:https://issues.redhat.com/browse/OCPBUGS-31469[*OCPBUGS-31469*])
+* Previously, when users imported image stream tags, ICSP (`ImageContentSourcePolicy`) was not allowed to co-exist with IDMS (`ImageDigestMirrorSet`) and ITMS (`ImageTagMirrorSet`). {product-title} ignored any IDMS/ITMS created by the user and favored ICSP. With this release, they are allowed to co-exist since importing image stream tags will now respect IDMS/ITMS when ICSP is also present. (link:https://issues.redhat.com/browse/OCPBUGS-31469[*OCPBUGS-31469*])
```

Comment:
Should these be the other way around?

```suggestion
* Previously, when users imported image stream tags, `ImageContentSourcePolicy` (ICSP) was not allowed to co-exist with `ImageDigestMirrorSet` (IDMS) and `ImageTagMirrorSet` (ITMS). {product-title} ignored any IDMS/ITMS created by the user and favored ICSP. With this release, they are allowed to co-exist since importing image stream tags will now respect IDMS/ITMS when ICSP is also present. (link:https://issues.redhat.com/browse/OCPBUGS-31469[*OCPBUGS-31469*])
```

Reply:
You're right, I made the change. Thanks!